core/api/v1/keytransparency.proto

```diff
@@ -264,7 +264,7 @@ message ListMutationsResponse {
 //    that user, and signatures indicating who made the last change.
 //
 // Each User also has a collection of historical values for user:
-// `/v1/directories/*/users/*/history`
+// `/v1/history/directories/*/users/*`
```

Comment:
do you think history belongs underneath directories? Would love to hear your thoughts on pro/cons/

Reply:
I don't see much difference between the two, but I'm also not a resource-oriented design expert.

What about a third option - that :listHistory is a custom verb on the directory+user resource?

Reply:
I'm not an expert either. Taking a hint from Google Drive's [API](https://developers.google.com/drive/api/v3/reference/revisions), revisions seems like it should come after `user_id`, which, as you say is screwing us a bit in that it means we can't have `user_id` as the last URI segement. 

* Get a user's current public key
`/v1/directories/{directory_id}/users/{user_id}`
* Get a user's public key at a specific revision
`/v1/directories/{directory_id}/users/{user_id}/revisions/{revision_id}/`
* List a user's public keys through time. 
`/v1/directories/{directory_id}/users/{user_id}/revisions/`